cms/models.py

```diff
@@ -30,10 +30,62 @@ class HomePage(Page):
     ]
 
     def get_context(self, request, *args, **kwargs):
-        return {
-            **super().get_context(request, *args, **kwargs),
-            "hero": self.hero,
-        }
+        context = super().get_context(request)
+        context.update({
```

Comment:
This is temporary data until we decide a specific way of implementing product cards backend.